Add `osu!catch World Cup 2025 concludes` news post (#13564)
ppy/osu-wiki: news/2025/2025-07-28-osucatch-world-cup-2025-concludes.md

---
layout: post
title: osu!catch World Cup 2025 Concludes
date: 2025-07-28 12:00:00 +0000
---

The osu!catch World Cup 2025 is officially over, concluding with the coronation of our brand-new champions.

![](/wiki/shared/news/2025-07-28-osucatch-world-cup-2025-concludes/banner.jpg)

44 teams entered the tournament, each carrying the hopes and dreams of their country's communities. Every week saw the elimination of teams more and more prestigious, often ending their journeys sooner than they had hoped. But for the select three who managed to carve their path to the very last round, a glorious podium finish awaited them.

Join us as we look back at this year's World Cup, through highlights and interviews from the ones who brought it to life!

## Summary

![Podium](/wiki/shared/news/2025-07-28-osucatch-world-cup-2025-concludes/podium.jpg)

> *I wake up, there is another coronation of glory...* Congratulations to ::{ flag=AR }:: Argentina who have been crowned our 2025 osu!catch champions for the first time in history! This is not only Argentina’s first osu!catch title, but also their first win across all gamemodes, a historic achievement for the country. The reigning champions, the ::{ flag=US }:: United States, were defeated 7–2 in Grand Finals, finishing as runners-up. And finally, ::{ flag=FI }:: Finland secured a spot on the podium in third place, also a first for the country.

Reviewer: ```suggestion
*I wake up, there is another coronation of glory...* Congratulations to Argentina who have been crowned our 2025 osu!catch champions for the first time in history! This is not only Argentina’s first osu!catch title, but also their first win across all game modes, a historic achievement for the country. The reigning champions, the United States, were defeated 7–2 in Grand Finals, finishing as runners-up. And finally, Finland secured a spot on the podium in third place, also a first for the country.
```

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2511672807&t=00h14m32s&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

![](/wiki/shared/news/2025-07-28-osucatch-world-cup-2025-concludes/mappool.png)

This Grand Finals mappool was definitely one of the hardest ones we've ever seen in osu!catch history. With most maps having a very high mechanical requirement and diving deep into each specialized skill, even the remaining teams struggled. [-Rustyy](https://osu.ppy.sh/users/16355636)'s [Kyokugen BON Odori](https://osu.ppy.sh/beatmapsets/2402724#fruits/5210989) (NM2) is a good showcase of this, requiring players to hit brutal 230 BPM wiggles alongside heavy tech patterns. For other picks, extremely high stamina was demanded of the players–[Nidoking](https://osu.ppy.sh/users/7034422)'s [Angel dust (2016 Radio Mix)](https://osu.ppy.sh/beatmapsets/2402805#fruits/5211177) featured constant wiggles at a high level of precision and on low approach rate for two ruthless minutes.

> For other slots, players got what they were used to. The classic [Daletto](https://osu.ppy.sh/users/7592136) Hard Rock stream map appeared in the form of [Pain Remains I: Dancing Like Flames](https://osu.ppy.sh/beatmapsets/2402732#fruits/5211006) (HR1), as well as the typically unorthodox [ExGon](https://osu.ppy.sh/users/214187) Double Time control pick with [MYS](https://osu.ppy.sh/beatmapsets/2402781#fruits/5211124).osu!mania 7k World Cup 2022 Grand Finals' tiebreaker, [Duplicity Shade](https://osu.ppy.sh/beatmapsets/1698284#mania/3469849) was also featured as a convert pick in the pool, but as a [cut version](https://osu.ppy.sh/beatmapsets/2403136#fruits/5211965) (NM4), making for a hellish stack gauntlet that overwhelmed every player. 

Author: dt4 not nm4

> For other slots, players were greeted with some familiar faces. The classic [Daletto](https://osu.ppy.sh/users/7592136) Hard Rock stream map appeared in the form of [Pain Remains I: Dancing Like Flames](https://osu.ppy.sh/beatmapsets/2402732#fruits/5211006) (HR1), alongside the typically unorthodox [ExGon](https://osu.ppy.sh/users/214187) Double Time control pick with [MYS](https://osu.ppy.sh/beatmapsets/2402781#fruits/5211124) (DT4).osu!mania 7k World Cup 2022 Grand Finals' tiebreaker, [Duplicity Shade](https://osu.ppy.sh/beatmapsets/1698284#mania/3469849) was also featured as a convert pick in the pool, but as a [cut version](https://osu.ppy.sh/beatmapsets/2403136#fruits/5211965) (NM4). The result? A hellish stack gauntlet that overwhelmed every player. 

Reviewer: ```suggestion
For other slots, players were greeted with some familiar faces. The classic [Daletto](https://osu.ppy.sh/users/7592136) Hard Rock stream map appeared in the form of [Pain Remains I: Dancing Like Flames](https://osu.ppy.sh/beatmapsets/2402732#fruits/5211006) (HR1), alongside the typically unorthodox [ExGon](https://osu.ppy.sh/users/214187) Double Time control pick with [MYS](https://osu.ppy.sh/beatmapsets/2402781#fruits/5211124) (DT4).osu!mania 7k World Cup 2022 Grand Finals' tiebreaker, [Duplicity Shade](https://osu.ppy.sh/beatmapsets/1698284#mania/3469849) was also featured as a convert pick in the pool, but as a [cut version](https://osu.ppy.sh/beatmapsets/2403136#fruits/5211965) (NM4). The result? A hellish stack gauntlet that overwhelmed every player.
```

    </audio>
</div>

[WangleLine](https://osu.ppy.sh/beatmaps/artists/427) made her appearance earlier in the tournament with [Island of Orchids](https://osu.ppy.sh/beatmapsets/2384624#fruits/5156307) being used in the Round of 32's mappool. Since then, she's been officially welcomed as a [Featured Artist](https://osu.ppy.sh/home/news/2025-07-13-new-featured-artist-wangleline), and this time her music returned to the mappool as an osu!original with [Goodbye To Your Vicious Cycles](https://osu.ppy.sh/beatmapsets/2402735#fruits/5211009) (HD3), mapped by [Phob](https://osu.ppy.sh/users/6069462) and [wwwww](https://osu.ppy.sh/users/8434466). This dark and gritty neuro drum and bass song was the perfect match for them both, who turned it into one of the most terrifying Hidden picks to date. Extremely demanding on both the reading and mechanics, the map and the song are sure to leave players feeling like they went through a nightmare after playing it.

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/O_8FGiiad6U" frameborder="0" allowfullscreen></iframe>
</div>

> Here we are — the most anticipated part of every World Cup mappool reveal: the **Grand Finals tiebreaker**. And once again, it absolutely delivered. [Yooh](https://osu.ppy.sh/beatmaps/artists/218) was back with another World Cup tiebreaker, this time in the form of [Eternity](https://osu.ppy.sh/beatmapsets/2402789#fruits/5211146). Mapped through a massive collaborative effort by [Zyzyx](https://osu.ppy.sh/users/2888013), [autofanboy](https://osu.ppy.sh/users/636114), [Bunnrei](https://osu.ppy.sh/users/829284), [-Ken](https://osu.ppy.sh/users/4430811), and [wwwww](https://osu.ppy.sh/users/8434466), this beast of a project served as the perfect culmination of everything this tournament had been building toward. 

Reviewer: ```suggestion
Here we are — the most anticipated part of every World Cup mappool reveal: the **Grand Finals tiebreaker**. And once again, it absolutely delivered. [Yooh](https://osu.ppy.sh/beatmaps/artists/218) was back with another World Cup tiebreaker, this time in the form of [Eternity](https://osu.ppy.sh/beatmapsets/2402789#fruits/5211146). Mapped through a massive collaborative effort by [Zyzyx](https://osu.ppy.sh/users/2888013), [autofanboy](https://osu.ppy.sh/users/636114), [Bunnrei](https://osu.ppy.sh/users/829284), [-Ken](https://osu.ppy.sh/users/4430811), and [wwwww](https://osu.ppy.sh/users/8434466), this beast of a project served as the perfect culmination of everything this tournament had been building toward.
```

Featuring several original character designs from illustrators [Kyuare](https://osu.ppy.sh/users/2705312), [HiROdoge](https://osu.ppy.sh/users/15251809), and [Xiaoraini](https://osu.ppy.sh/users/3607337), brought to life through the graphic design work of [AlexDunk](https://osu.ppy.sh/users/9194799), [Boolmaster Flex](https://osu.ppy.sh/users/5394681), and [hourius](https://osu.ppy.sh/users/12580298), all combined in the mind-blowing storyboard by [Kuroise](https://osu.ppy.sh/users/14547127), [Noffy](https://osu.ppy.sh/users/1541323), and [Coppertine](https://osu.ppy.sh/users/7279762) — this map hides a mountain of lore and secrets. There's more to experience than words can convey. We strongly urge you to go check it out for yourself!

---

> 

Reviewer: needs a banner asset here, ill provide it in the thread

You can check out the previews for each one of them right below, and the maps by clicking on the banners!

### Round of 16

[![](/wiki/shared/news/2025-06-27-osucatch-world-cup-2025-round-of-16-recap/Planetary_Exclusion_Zone_banner.png)](https://osu.ppy.sh/beatmapsets/2388185#fruits/5168126)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/482/Songs/XenjeS - Planetary Exclusion Zone.mp3">
    </audio>
</div>

### Quarterfinals

[![](/wiki/shared/news/2025-07-05-osucatch-world-cup-2025-quarterfinals-recap/Heliosphere_Banner.png)](https://osu.ppy.sh/beatmapsets/2391836#fruits/5177897)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/303/Songs/Vorso_-_Heliosphere.mp3">
    </audio>
</div>

[![](/wiki/shared/news/2025-07-28-osucatch-world-cup-2025-concludes/Symholic_Banner.jpg)](https://osu.ppy.sh/beatmapsets/2391791#fruits/5177767)

<div class="osu-md__paragraph">
    <audio controls>

>         <source src="https://assets.ppy.sh/artists/303/Songs/Vorso_-_Heliosphere.mp3">

Reviewer: ```suggestion
        <source src="https://assets.ppy.sh/artists/130/Songs/Symholic_feat._Rina_Komatsu_-_Veil_of_Ash.mp3">
```

## Grand Finals recap

### ::{ flag=AR }:: Argentina vs. ::{ flag=US }:: United States

**[View the match information here!](https://osu.ppy.sh/community/matches/118732915)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&time=1h5m4s&video=2517677050" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

It was now time for the long-awaited Grand Finals. Argentina (#2) and the United States (#1), the two clear favourites, standing head and shoulders above the rest, would face off once again. One year ago, Argentina was the one coming out of the losers' bracket in the ultimate match, only to lose 7–4. This time, however, Argentina had sent the United States to the losers' bracket, defeating them in the winners' bracket final 7–4. After a decisive 7–0 victory over Finland, it was now time for the rematch — of both last year's grand final and this year's winners' final.

> In their previous match, most of the mappool didn't really favour one team over the other, with both teams contesting each other on most picks — it mainly came down to match performance. This time, however, with the continuous rise in mappool difficulty, and especially mechanical difficulty, it was unclear whether the United States could keep up with the Argentinian roster. All of their core players are well-known within the community for their prowess on fast and stamina-draining maps, which can't really be said about the American roster as a whole, whose strengths mainly lie in other skills. 

Reviewer: ```suggestion
In their previous match, most of the mappool didn't really favour one team over the other, with both teams contesting each other on most picks — it mainly came down to match performance. This time, however, with the continuous rise in mappool difficulty, and especially mechanical difficulty, it was unclear whether the United States could keep up with the Argentinian roster. All of their core players are well-known within the community for their prowess on fast and stamina-draining maps, which can't really be said about the American roster as a whole, whose strengths mainly lie in other skills.
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2705312" style="background-image: url('/wiki/shared/avatars/Kyuare.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/2705312">Kyuare</a></p>

The moment I heard the song, I knew we were about to create banger designs. Many ideas surged into my mind, and I had to start sketching them out right away to capture the initial creative momentum.

![](/wiki/shared/news/2025-07-28-osucatch-world-cup-2025-concludes/eternitydesign1.jpg)

My strength really shines during the conceptual phase, where I excel at interpreting abstract instrumental tracks without lyrics, and then transforming them into cool visual ideas. Yooh granted a lot of creative freedom, so it was important for our design team to get a strong direction right from the start to keep everyone's ideas organized. I created multiple sketches based on our initial team discussion, and I'm glad those sketches were able to set a nice foundation for us to build on.

> Later during production, I further expanded the sketches into more visual references, helping everyone understand the intended mood and narrative direction while still leaving space for individual creativity. 

Reviewer: ```suggestion
Later during production, I further expanded the sketches into more visual references, helping everyone understand the intended mood and narrative direction while still leaving space for individual creativity.
```

![](/wiki/shared/news/2025-07-28-osucatch-world-cup-2025-concludes/eternitydesign2.jpg)

> This created a great synergy. Everyone felt empowered to contribute, which elevated both the ambition and cohesion of the project. The result was a team-wide focus on storytelling, and a lore that felt genuinely alive.

Reviewer 2: ```suggestion
My strength really shines during the conceptual phase, where I excel at interpreting abstract instrumental tracks without lyrics, and then transforming them into cool visual ideas. Yooh granted a lot of creative freedom, so it was important for our design team to get a strong direction right from the start to keep everyone's ideas organised. I created multiple sketches based on our initial team discussion, and I'm glad those sketches were able to set a nice foundation for us to build on.
```

Author: kinda against editing the interview like this even if it's technically a bit more coherent, would rather keep it as it is to preserve the interviewee's intention

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/14547127">Kuroise</a></p>

I've always loved Yooh's works. My favorite World Cup tiebreaker is [RPG](https://osu.ppy.sh/beatmapsets/1633250#osu/3333745) after all. When I got a chance to participate in this project, I was beyond ecstatic. Yooh's design suggestions leaned towards angels, battle and salvation. Like Kyuare mentioned, Yooh gave us a lot of liberty in designing a character that fits the song. Although he didn't ask for the designs to be focused on storytelling, me and Kyuare wanted to create a story for the song. We envisioned a story about an immortal queen once respected for her wisdom, who falls into corruption and grief as her kingdom crumbles and she becomes a shadow of her former self. After Kyuare sketched out our main character Lucine, I started planning what the general narrative of each song section would be. With Noffy also collaborating on the storyboard, we were able to plan out most of the story during the first week of production.

> I also decided to have the lore encrypted throughout the video and storyboard to build a more mysterious feeling. That's why we created Lucine Crypt, a fully decodable font. 

Reviewer: ```suggestion
I also decided to have the lore encrypted throughout the video and storyboard to build a more mysterious feeling. That's why we created Lucine Crypt, a fully decodable font.
```

This project wasn't just a design effort. We also had [autofanboy](https://osu.ppy.sh/users/636114), [Bunnrei](https://osu.ppy.sh/users/829284), [-Ken](https://osu.ppy.sh/users/4430811), [wwwww](https://osu.ppy.sh/users/8434466), and [Zyzyx](https://osu.ppy.sh/users/2888013) working tirelessly on the map, with [fayew](https://osu.ppy.sh/users/12498861) doing hitsounding and [qwhj79](https://osu.ppy.sh/users/7547506) contributing to the replay. It truly felt like a team accomplishment across all staff members. Honestly, if even one person had been missing, I don't think this project would have been possible.

</div>

---

> 

Reviewer: also needs banner

Author: added

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/6567640" style="background-image: url('/wiki/shared/avatars/Realmaas.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-loved" href="https://osu.ppy.sh/users/6567640">Realmaas</a></p>

After this year's win, motivation seems to be at an all-time high in the Argentinian community. What can we expect from your team in the future?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8172283" style="background-image: url('/wiki/shared/avatars/-Story-.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/8172283">-Story-</a></p>

I'm very happy with everyone who congratulated us, and I'm not just talking about Argentina when I say that we truly received a LOT of support. It's undoubtedly a great memory for us, and it means a lot, even though there's some support and also hate but we choose to focus on the effort it took to get here, day after day.

> None of this would have been achieved without the motivation we had as a team, as a family, as a friendship. We went through a lot but I would like to thank the friends i have as colleagues. My best friends are [Zeus](https://osu.ppy.sh/users/8329151), [Alex](https://osu.ppy.sh/users/17738253), [VelperK](https://osu.ppy.sh/users/348467), [Naarkz](https://osu.ppy.sh/users/19490380), [Gonza](https://osu.ppy.sh/users/17309439) and I want to say thanks for everything to them and for the supporters that were there every single match and day to pray for our win ❤️ 

Reviewer: ```suggestion
None of this would have been achieved without the motivation we had as a team, as a family, as a friendship. We went through a lot but I would like to thank the friends i have as colleagues. My best friends are [Zeus](https://osu.ppy.sh/users/8329151), [Alex](https://osu.ppy.sh/users/17738253), [VelperK](https://osu.ppy.sh/users/348467), [Naarkz](https://osu.ppy.sh/users/19490380), [Gonza](https://osu.ppy.sh/users/17309439) and I want to say thanks for everything to them and for the supporters that were there every single match and day to pray for our win ❤️
```

Now... Regarding the Catch World Cup, I think it will depend on several factors that allow us to have or make time to play them, but we no longer have the pressure of being champions because we already are. Having fun and accepting that the game is a game is the key to understand that you don't have to take things too seriously and that no matter how much people judge you, whether you win or lose, it doesn't matter! You have to enjoy having teammates, moments where you fall and get up, situations where you don't feel like you're what you demonstrated in practice, and much more.

Take the World Cup as a place to explore a more competitive and healthy environment where you can have fun and play with your friends.
Don't overthink too much and enjoy what this is, just a game that we all love.

> See you next year, I'm hoping to play with all my friends again and again because they are the ones that motivate me to keep playing. 

Reviewer: ```suggestion
See you next year, I'm hoping to play with all my friends again and again because they are the ones that motivate me to keep playing.
```

Will this be the start of a new Argentinian dynasty in osu!catch World Cups? Only the future can tell, but this could mark a turning point in the tournament's history.

## Mappool recap

**[Download the mappool here! (207 MB)](https://packs.ppy.sh/P293%20-%20osu!catch%20World%20Cup%202025%3A%20Grand%20Finals.zip?1752690501)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2511672807&t=00h14m32s&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

![](/wiki/shared/news/2025-07-28-osucatch-world-cup-2025-concludes/mappool.png)

> This Grand Finals mappool was definitely one of the hardest ones we've ever seen in osu!catch. With most maps having a very high mechanical requirement and diving deep into each specialized skill, even the remaining teams struggled. [-Rustyy](https://osu.ppy.sh/users/16355636)'s [Kyokugen BON Odori](https://osu.ppy.sh/beatmapsets/2402724#fruits/5210989) (NM2) is a good showcase of this, requiring players to hit brutal 230 BPM wiggles alongside heavy tech patterns. For other picks, extremely high stamina was demanded of the players–[Nidoking](https://osu.ppy.sh/users/7034422)'s [Angel dust (2016 Radio Mix)](https://osu.ppy.sh/beatmapsets/2402805#fruits/5211177) featured constant wiggles at a high level of precision and on low approach rate for two ruthless minutes.

Reviewer 2: ```suggestion
This Grand Finals mappool was definitely one of the hardest ones we've ever seen in osu!catch history. With most maps having a very high mechanical requirement and diving deep into each specialized skill, even the remaining teams struggled. [-Rustyy](https://osu.ppy.sh/users/16355636)'s [Kyokugen BON Odori](https://osu.ppy.sh/beatmapsets/2402724#fruits/5210989) (NM2) is a good showcase of this, requiring players to hit brutal 230 BPM wiggles alongside heavy tech patterns. For other picks, extremely high stamina was demanded of the players–[Nidoking](https://osu.ppy.sh/users/7034422)'s [Angel dust (2016 Radio Mix)](https://osu.ppy.sh/beatmapsets/2402805#fruits/5211177) featured constant wiggles at a high level of precision and on low approach rate for two ruthless minutes.
```
